nemo/nemo/backends/pytorch/actions.py

```diff
@@ -971,8 +974,8 @@ def _check_tuples(list_of_tuples):
 
     def _get_all_modules(
             self, training_loop, callbacks, logging_callchain=None):
-        """Gets all neural modules that will be used by train() and eval() via
-        EvaluatorCallbacks. Saves all modules to self.modules
+        """Gets all neural module that will be used by train() and eval() via
```

Comment:
In this docstring, first two should be plural, last one depends on the name of the variable but I am guessing not.